src/action.ts

```diff
@@ -393,8 +394,16 @@ class Action {
     + "<td>Result</td></tr></thead><tbody>"
 
     for (const task of report.actions) {
-      task.passed ? tasksPassed++ : tasksFailed++
```

Comment:
If the `tasksSkipped` is not used to be incremented, I do not see the difference between replacing this line with that if-block.